go.mod

```diff
@@ -269,7 +269,7 @@ replace (
 	github.com/golangplus/testing => github.com/golangplus/testing v0.0.0-20180327235837-af21d9c3145e
 	github.com/google/btree => github.com/google/btree v1.0.1
 	github.com/google/cadvisor => github.com/google/cadvisor v0.43.0
-	github.com/google/cel-go => github.com/google/cel-go v0.9.0
+	github.com/google/cel-go => github.com/google/cel-go v0.9.1-0.20220218004315-9ef90eddf66e
```

Comment:
will there be a tagged release for us to use before 1.24?

Reply:
Yes, we should be able to push a new CEL release with time to spare - tagging @jpbetz here to confirm that, though.

Reply:
I checked with @TristonianJones and it sounds like should be able to have a tagged version number. We're planning to land the last outstanding CEL changes today and then we'll tag and open the cel-go version bump PR against k/k.